src/idex/identity-resolver.js

```diff
@@ -74,6 +69,8 @@ export function IdentityResolver (config, storageHandler) {
     const tuples = []
     tuples.push(['duid', encodedOrNull(nonNullConfig.peopleVerifiedId)])
     tuples.push(['us_privacy', encodedOrNull(nonNullConfig.usPrivacyString)])
+    tuples.push(['gdpr', encodedOrNull(nonNullConfig.gdprApplies)])
```

Comment:
Yeah this won't work here... gdpr will be set to false which means it's false and you won't even send that info. Plus, the gdprApplies in the config is boolean, however the string must be 0|1 from what i read.

Reply:
I mixed up the values that are passed into live-connect. I'll update the prebid and live-connect prs.

Reply:
updated https://github.com/LiveIntent/live-connect/pull/26/commits/8e9aa5fe841b8d9be33eb33fa04132d1b10693c7